### ISSUE TITLE
generic turn on lights flow

### ISSUE THREAD

**Issue by idarlund** [athombv/homey#2280]
**What did you try to achieve?**
Create a generic "if motion is detected in a ZONE, then turn on lights in that ZONE".

**The current situation.**
Right now we have to create a flow for each zone like this:
![image](https://user-images.githubusercontent.com/3417634/36903001-9e7ca8ee-1e2c-11e8-8966-5b552fb1ea5b.png)

**How and why the current situation is problematic.**
With many zones, this will generate a lot of flows.

**A detailed proposal or pull request that demonstrates how the problem could be solved.**
This could be resolved by allowing the "zone" variable to be added to "Turn specific devices on", like this:
![image](https://user-images.githubusercontent.com/3417634/36903101-eb2b42fe-1e2c-11e8-95c4-681326b59557.png)

That is allready possible with a generic push message on motion detected:
![image](https://user-images.githubusercontent.com/3417634/36903154-1fdc2428-1e2d-11e8-8e55-b14bdfc71d1f.png)

**A use case – who needs this feature and why?**
This feature will limit the amount of "turn on lights here and there" flows, and instead make it neat. It's also easier to create these generic events instead of those specific ones. Also when adding a new motion detector and lights into a zone, it will start working without having to create another specific flow.

**Comment by idarlund** [comment]
Updated headline; this feature can be used for much more than just turning on lights. This feature can be used to open curtains, turn on fans, etc. dynamically in the zone where something triggered.

**Comment by angr76** [comment]
![screen shot 2018-04-09 at 15 06 53](https://user-images.githubusercontent.com/37574543/38500683-6c2b2382-3c0b-11e8-8843-0416f311fd03.png)

I also discovered that this is not available. Would really like to see this implemented as well.  Big potential use-cases!

**Comment by idarlund** [comment]
have this feature request been considered yet? I see that "needs discussion" has been removed; does that mean that this feature request has been approved?

**Comment by WeeJeWel** [comment]
Definitely. We will introduce a new feature called 'Active Zones' soon. When a sensor detects something in a zone, this zone is marked as active, and then devices can turn on automatically. There'll also be a trigger like 'This zone has been inactive for <x> minutes', to solve the timer problem.

**Comment by idarlund** [comment]
Awsome! That makes it even easier in the flows! Any eta or roadmaps on that feature that we can follow?

**Comment by WeeJeWel** [comment]
Most likely v2.1 :)
